api/ai/interfaces/algorithm_options.py

```diff
@@ -247,13 +249,10 @@ def get_schema() -> Schema:
 
 @dataclass
 class MultipleRoundRobinAlgorithmOptions(AlgorithmOptions):
-    projects: List[Project]
+    utility_function: Callable[[Student, TeamShell], float]
```

Comment:
Utility functions seem like config things to me, unless it's intended that they change per call of the algorithm?

Reply:
Sounds right, it's a config